openrouteservice-api-tests/src/test/java/org/heigit/ors/services/routing/ResultTest.java

```diff
@@ -1105,9 +1102,9 @@ private void checkExtraConsistency(Response response) {
 				amount += jSummaryValues.getDouble("amount");
 			}
 
-			Assert.assertEquals(Math.abs(routeDistance - distance) < 0.5, true);
+			Assert.assertTrue(Math.abs(routeDistance - distance) < 0.5);
 
-			Assert.assertEquals(Math.abs(amount - 100.0) < 0.1, true);
+			Assert.assertTrue(Math.abs(amount - 100.0) < 0.1);
```

Comment:
Both statements look like they could use `assertEquals(double expected, double actual, double delta)` instead?

Reply:
ignoring, it's a V1 test anyway